doc/psidk/src/run-infrastructure/cli/psinode.md

```diff
@@ -77,6 +77,10 @@ psinode - The psibase blockchain server
   - `bearer`:*key*: Allows access with a bearer token, which must be sent in the HTTP `Authorization` header. The key is an arbitrary string which will be used to verify the tokens. Tokens can be generated by `psibase create-token` or by the `/native/admin/login` endpoint.
   This option may be specified more than once. A client can access the admin API if it satisfies any of the conditions.
 
+- `--http-timeout` *seconds*
+
+  tells psinode how long to wait before closing an idle connection.
```

Comment:
I think here would be the right place to mention that the units can be specified as: nothing (seconds), s, ms, us/μs, or ns